### ISSUE TITLE
Error on araig_calculator when launching turtlebot3_sim_test

### ISSUE THREAD

**Issue by IPA-KUT-CL** [ipa320/araig_test_stack#76]
Hi,

When I launch a test by `roslaunch turtlebot3_sim_test test1_braking.launch`, errors occurred as:

>   File "/home/chang/catkin_ws/src/araig_test_stack/araig_calculators/scripts/folder_bagger_node", line 5, in <module>
>     from loggers.folder_bagger import FolderBagger
> ImportError: No module named folder_bagger
> Traceback (most recent call last):
>   File "/home/chang/catkin_ws/src/araig_test_stack/araig_calculators/scripts/rosparam_logger_node", line 4, in <module>
>     from loggers.rosparam_logger import RosparamLoggerClass
> ImportError: No module named rosparam_logger
> Traceback (most recent call last):
>   File "/home/chang/catkin_ws/src/araig_test_stack/araig_calculators/scripts/results_logger_node", line 5, in <module>
>     from loggers.results_logger import ResultsLoggerClass
> ImportError: No module named results_logger

Therefore, processed as `folder_bagger_node-10`, `param_logger_node-11`, `results_logger_node-12` and `robot_speed_logger-13` died.

I think there might be something wrong about linking the node script with source file but don't know how to amend this.

Regards,
Chang

**Comment by ipa-kut** [comment]
Yeah the SIM test has not been curated in a while, I can believe these errors. I am not sure if/when we can look into this though.
